content/ja/project-management.md

```diff
@@ -3,21 +3,22 @@ author: イソリア
 date: 2015-05-09T00:05:19.000Z
 publishdate: 2012-01-10T00:05:19.000Z
 draft: false
-title: プロジェクト マネージメント
+title: プロジェクト管理
 slug: project-management
-subtitle: 成功する為のコーディネーション
+subtitle: 成功する為のPM
 description: イソリアから - 東京およびアジア太平洋地域の実施しているプロジェクトで成功を収めています。私たちのPMアプローチは実践的で独自性があります。
 weight: 37
 alternatelocales:
   - en-US
 tags:
   - プロジェクト管理
+  - プロジェクトマネージメント
   - コーディネーション
   - 人 金 物
   - オフィス移転
   - UPS
   - テクノロジーリフレッシュ
-  -
+  -　PM
```

Comment:
@eishikawa you broke the website again with a Japanese space. Please use English characters and spaces only for these things. I will fix.